tests/tests/test_deactivate_expired_reasons.py

```diff
@@ -74,3 +74,21 @@ def test_command_should_not_anonymize_customer_if_expiring_reason_is_not_related
         self.assertAnonymizedDataNotExists(self.customer, 'first_name')
         assert_not_equal(self.customer._anonymize_obj(), CUSTOMER__FIRST_NAME)
         self.assertAnonymizedDataExists(self.customer, 'first_name')
+
+    def test_facebook_purpose_should_anonymize_customer_with_facebook_id(self):
+        customer = Customer.objects.get(pk=self.customer.pk)
+        legal_reason = LegalReason.objects.create_consent(FACEBOOK_SLUG, customer)
+        legal_reason.save()
+        legal_reason.expire()
+        customer.refresh_from_db()
+        assert_not_equal(customer.first_name, CUSTOMER__FIRST_NAME)
+
+    def test_facebook_purpose_should_anonymize_customer_without_facebook_id(self):
```

Comment:
Why? I would expect it will not anonymize...

Reply:
Why? I want to test it so, I created specific anonymizer for it. Which has set this logic.